dotcom-rendering/src/web/components/Island.tsx

```diff
@@ -2,7 +2,7 @@
 
 import { Placeholder } from './Placeholder';
 
-type When = 'idle' | 'visible';
+type When = 'idle' | 'visible' | 'interaction';
```

Comment:
Should we add this here? Perhaps, now that we have different use cases for different 'whens', we could remove the entire `When` type and just inline `'idle' | 'visible'`?